config/snapshotgc/snapshotgc.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: snapshot-garbage-collector
+spec:
+  schedule: "0 11 * * *" # every day at 11AM UTC
```

Comment:
Suggestion: maybe we can run this at a lower-traffic time, somewhere between 0-6AM UTC.

Reply:
done